flink-connector/flink/src/test/java/org/apache/gravitino/flink/connector/integration/test/FlinkCommonIT.java

```diff
@@ -76,32 +95,20 @@ public void testCreateSchema() {
   }
 
   @Test
+  @EnabledIf("supportSchemaOperation")
   public void testGetSchema() {
```

Comment:
rename to `testGetSchemaWithoutCommentAndOption`? and could you place `testGetSchemaWithCommentAndOptions` after this test?